Hymns/Display/DisplayHymnBottomBarViewModel.swift

```diff
@@ -27,14 +28,15 @@ class DisplayHymnBottomBarViewModel: ObservableObject {
          hymnsRepository repository: HymnsRepository = Resolver.resolve(),
          mainQueue: DispatchQueue = Resolver.resolve(name: "main"),
          backgroundQueue: DispatchQueue = Resolver.resolve(name: "background"),
-         systemUtil: SystemUtil = Resolver.resolve()) {
+         systemUtil: SystemUtil = Resolver.resolve(), workingURL: URL? = nil) {
```

Comment:
I generally don't like passing parameters around because if added 1-2 parameters for every new feature, the app would get unwieldy very quickly. Can you think of a better way? Maybe we can use the repository and fetch the English song version asynchronously if the current mp3 doesn't exist.